p2p/pex/pex_reactor.go

```diff
@@ -264,6 +264,8 @@ func (r *Reactor) Receive(chID byte, src Peer, msgBytes []byte) {
 			r.SendAddrs(src, r.book.GetSelectionWithBias(biasToSelectNewPeers))
 			go func() {
 				// In a go-routine so it doesn't block .Receive.
+				// Sleep for one min to allow the peer to handle the addrs.
+				time.Sleep(1 * time.Minute)
```

Comment:
Will this effectively allow peers to send multiple `pexRequestMessage`s during the 1 minute time interval, potentially spawning multiple goroutines that will attempt to stop the same peer? If so, what's the impact of that on the system?

Reply:
Thanks for asking, now that I've looked into it, I think the situation is the same as before, and that only on "stop goroutine" will be spawned per peer, see the check a bit above at https://github.com/tendermint/tendermint/pull/4467/files#diff-8c6b1a350a161ced01d68514ccda7484R256